src/main/scala/cromwell/engine/backend/jes/JesBackendCall.scala

```diff
@@ -113,4 +114,9 @@ class JesBackendCall(val backend: JesBackend,
       Option(Map("log" -> WdlFile(jesLogGcsPath)))
     )
   }
+
+  override val pollingInitialInterval: FiniteDuration = 20.seconds
+  override val pollingMaxInterval: FiniteDuration = 10.minutes
+  override val pollingMultiplier: Double = 1.1D
+  override val pollingInitialGap: FiniteDuration = 1.minute
```

Comment:
I thought this was going to be a case class?
`case class PollingSpec(pollingInitialInterval: FiniteDuration, pollingMaxInterval: FiniteDuration,pollingMultiplier: Double, pollingInitialGap: FiniteDuration)`?